### PR TITLE
Grt: Fixing the overlapped edges after maze3D

### PR DESCRIPTION
**Pull request by luis201420**
Fixing overlapping edges generated after the maze3D function

### CODE REVIEW

**Comment by maliberty** [comment]
Have you run a secure CI?

**Review by eder-matheus — CHANGES_REQUESTED** [review]
Before any other changes I requested, please resolve the failures in the unit tests. It's important to have the unit tests working to ensure that your further changes will not break anything.

File: src/grt/src/fastroute/include/FastRoute.h [hunk 1]
```diff
@@ -328,6 +323,13 @@ class FastRouteCore
 
   // maze3D functions
   void mazeRouteMSMDOrder3D(int expand, int ripupTHlb, int ripupTHub);
+  void addNbrPoints(int netID,
```
Comment (eder-matheus) [inline src/grt/src/fastroute/include/FastRoute.h:326]: Avoid abbreviations in the function names. Change `addNbrPoints` to `addNeighborPoints`.

**Review by eder-matheus — APPROVED** [review]
Just address my final comment before merging.
I see CI is passing.

**Pull request merged**